pydata_sphinx_theme/_templates/navbar-menu-buttons.html

```diff
@@ -0,0 +1,3 @@
+{%- block icon_links -%}
+{%- include "icon-links.html" with context -%}
```

Comment:
Should we put the content of `icon-links.html` simply here?

Reply:
you mean and remove the `block`? I think the main benefit is if somebody wanted to override that section and add something of their own, do we want this to be possible?

Reply:
we currently override `block icon_links` in the MNE-Python site. So removing it would necessitate some changes for us. See https://github.com/mne-tools/mne-python/blob/main/doc/_templates/docs-navbar.html

Reply:
I didn't mean to remove the `block`, but just the content of the file could directly be put in this template file instaed of using the `include`?

Also, to keep to our terminology of "icon links", should we call this "navbar-menu-icons"? instead of "buttons"?

Reply:
@drammock I *think* that also with the current change, you will need to do a small update to your custom template. Because the `{%- block icon_links -%}` is now no longer defined in docs-navbar.html, but in this navbar-menu-buttons.html which gets inserted into docs-navbar.html.

(now, I don't know enough of sphinx / jinja to know in which order what happens)

Reply:
thanks for the heads-up.  I think we're pinned at the moment, but I'll make the necessary changes and unpin once this PR is merged.